### PR TITLE
[DX-1437] modell --> model, flavour --> flavor, categorise --> categorize, licence --> license, normalise --> normalize, programme --> program, recognise --> recognize

### PR DESCRIPTION
**Pull request by Roeegg2**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/content/tyk-developer-portal.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ aliases:
 
 The Tyk Developer Portal enables you to expose a facade of your APIs and then allow third-party developers to register and use your APIs.
 
-The Tyk Developer Portal comes into two flavours:
+The Tyk Developer Portal comes into two falvors:
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-developer-portal.md:15]: Fix typo

Comment (dcs3spp) [inline tyk-docs/content/tyk-developer-portal.md]: Fix falvors typo

Comment (letzya) [inline tyk-docs/content/tyk-developer-portal.md]:
```suggestion
The Tyk Developer Portal comes into two flavors:
```

**Review by letzya — APPROVED** [review]

**Pull request merged**